operators/Tane/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Tané",
```

Comment:
Hey this might fail on our validation as we don't allow special characters `é` right now - is it possible to just use `e` here and in description? @tksohishi

Reply:
OK, will update them.